lib/map-blueprint.js

```diff
@@ -12,12 +14,15 @@ export default class MapBlueprint {
                81, 78, 78, 78, 78, 78, 78, 92,
                168, 164, 164, 164, 164, 164, 164, 165
              ],
+      barriers: this.barriers,
       columns: 8,
       rows: 8,
       blueFlagOptions: { color: 'blue', x: 150, y: 150, tsize: this.tsize },
       redFlagOptions: { color: 'red', x: 250, y: 250, tsize: this.tsize },
-      bluePlayerOptions: { x: 150, y: 150, color: 'blue', controls: 'wasd' },
-      redPlayerOptions:  { x: 100, y: 100, color: 'red', controls: 'arrows' }
+      bluePlayerOptions: { x: 150, y: 150, color: 'blue', 
+                           controls: 'wasd', acceleration: 0.1 },
+      redPlayerOptions:  { x: 100, y: 100, color: 'red', 
```

Comment:
Yeah, I thought maybe if we changed to an attribute of player (e.g. We made power ups that increased/decreased acceleration) then it should be defined. But in that case we should probably just default it 0.1 and not define it in the blueprint. Premature optimization ...